### DISCUSSION TITLE
Custom lambda schedule in RFE?

### DISCUSSION THREAD

**Discussion by charlie-athae** [OpenFreeEnergy/openfe#941] [General]
Hi, I was trying to get a custom lambda schedule in RFE and noticed that the LambdaProtocol class allows a custom lambda_schedule array of floats.  However, it doesn't seem like this isn't read anywhere in the RFE paths, and the lambda_settings don't seem to expose this in the json files.  Is there a way to set a manual lambda schedule rather than the default linear spaced ones for relative free energies?  Thanks!

**Comment by dotsdl** [comment]
Hi @charlie-athae, I think this one slipped past us! Sorry for the delayed reply.

@IAlibay any insights on this one from @charlie-athae's description?

In the meantime @charlie-athae, can you share a snippet as to how you've tried setting the `lambda_schedule`? Seeing how you're attempting to do this may help us to identify how best to help you.

**Comment by charlie-athae** [comment]
Hi, I haven't actually worked on this in a while so I don't have the code I was playing with anymore, but I think it would just be the ability to access custom lambda schedules if I want better sampling in the free energy calculation similar to how its done in the AFE test cases:
openfe/openfe/tests/protocols/test_openmm_afe_solvation_protocol.py (test_validate_lambda_schedule_nwindows)

It'd be nice if the RFE code allowed the ability to also set a lambda schedule.

In test_openmm_equil_rfe_protocols.py, the lambda_schedule parameter can be read in but it isn't used in favor of the LambdaProtocol function being read to generate the schedule.
